.github/workflows/netlify.yml

```
@@ -0,0 +1,60 @@
+name: Visual regression tests with Netlify
+
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
+
+    if: |
+      ( github.repository == 'optuna/optuna-visualization-regression-tests' ) &&
+      (
+         contains(github.event.pull_request.labels.*.name, 'safe-to-run-visual-regression-tests') ||
+         ( github.event_name == 'workflow_dispatch' )
+      )
+
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+
+    - name: Install Optuna
+      run: |
+        pip install -U pip
+        pip install --progress-bar off .
+        git clone https://github.com/optuna/optuna.git
+        pip install optuna/
```

A nit, but I think we can simplify these lines.
```suggestion
        pip install git+https://github.com/optuna/optuna.git
```